core/src/View/Kokkos_ViewMapping.hpp

```diff
@@ -3351,6 +3351,7 @@ KOKKOS_FUNCTION bool within_range(Map const& map,
   return (((std::size_t)indices < map.extent(Enumerate)) && ...);
 }
 
+#ifndef KOKKOS_ENABLE_IMPL_MDSPAN
 template <class... Indices>
 KOKKOS_FUNCTION constexpr char* append_formatted_multidimensional_index(
```

Comment:
Given this one is a new function, can we do better interface (such as pass in an array instead of a `char*`)?  Or is it not worth it?

Reply:
this function isn't actually that old. That's part of the rework from Damien for error messages. std::array won't work since this is called on device.